src/WebUI/Components/DaemonSetListingComponent.tsx

```diff
@@ -28,7 +28,9 @@ export class DaemonSetListingComponent extends BaseComponent<IDaemonSetComponent
             <div>{
```

Comment:
i would recommend to do the filtering outside of the div, and do not show if the filter yields zero results.

items={(this.props.daemonSetList.items || []).filter((item)=>{
                        return Utils.filterByName(item.metadata,this.props.nameFilterKey);
                    }) }

Reply:
Moved filtering to outside div and returned only if there are elements to display.